### PR TITLE
Modify the header and font of app

### PR DESCRIPTION
**Pull request by Christian7sr**
The colors of the header were changed with those we defined for the
application. Also the structure of this one changed a little. We changed
the font of the application for a more comfortable for us, the font is
called "ubuntu".

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -1,11 +1,16 @@
-<header class="navbar navbar-fixed-top navbar-inverse">
+<header class="navbar navbar-fixed-top">
   <div class="container">
-    <%= link_to "Inicio", '#', id: "logo" %>
-    <nav>
+    <%= link_to "Inicio", root_path, id: "logo" %>
+    <nav class="barra">
+      <ul class="nav navbar-nav">
+        <li class="link"><%= link_to "Publicaciones", root_path,id:"link" %></li>
```
Comment (jschavesr) [inline app/views/layouts/_header.html.erb]: If you dont have the link. Use # instead

Comment (niquefaDiego) [reply app/views/layouts/_header.html.erb]: x2

File: app/assets/stylesheets/customs.scss [hunk 2]
```diff
@@ -1,9 +1,20 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
 
+/*colores
```
[the remark this answers is not in the document]
Comment (niquefaDiego) [reply app/assets/stylesheets/customs.scss]: x2

**Comment by niquefaDiego** [comment]
jajajajja ste men con esos nombres de colores XD

**Pull request merged**